core/store/migrationsv2/0010_bridge_fk.go

```diff
@@ -13,7 +13,7 @@ UPDATE pipeline_task_specs SET bridge_name = ts.json->>'name' FROM pipeline_task
 `
 	down10 = `
 ALTER TABLE pipeline_task_specs DROP CONSTRAINT fk_pipeline_task_specs_bridge_name;
-ALTER TABLE DROP COLUMN bridge_name;
+ALTER TABLE pipeline_task_specs DROP COLUMN bridge_name;
```

Comment:
I think you can do:

```
ALTER TABLE pipeline_task_specs DROP CONSTRAINT fk_pipeline_task_specs_bridge_name, DROP COLUMN bridge_name;
```

I believe postgres can do some internal magic to make things more efficient in some cases when you put it all in one statement.